src/main/java/net/tropicraft/core/common/item/TropicalMusicDiscItem.java

```diff
@@ -23,18 +24,13 @@ public TropicalMusicDiscItem(RecordMusic type, Properties builder) {
     @Override
     public void addInformation(ItemStack stack, World worldIn, List<ITextComponent> tooltip, ITooltipFlag flagIn) {
         super.addInformation(stack, worldIn, tooltip, flagIn);
-        tooltip.add(getDescLine(1).applyTextStyle(TextFormatting.GRAY));
+        tooltip.add(getDescLine(1).deepCopy().mergeStyle(TextFormatting.GRAY));
     }
     
     private ITextComponent getDescLine(int i) {
         return new TranslationTextComponent(this.getTranslationKey() + ".desc." + i);
     }
-    
-    @Override
-    public ITextComponent getRecordDescription() {
```

Comment:
Is there a replacement for this?

Reply:
Probably went about this the wrong way, but changed the `.desc.0` entries to be just `.desc` which is what gets defaulted to by vanilla.